internal/sdkv2provider/schema_cloudflare_custom_hostname.go

```diff
@@ -122,12 +122,12 @@ func resourceCloudflareCustomHostnameSchema() map[string]*schema.Schema {
 									Description:  fmt.Sprintf("Lowest version of TLS this certificate should support. %s", renderAvailableDocumentationValuesStringSlice([]string{"1.0", "1.1", "1.2", "1.3"})),
 								},
 								"ciphers": {
-									Type:     schema.TypeSet,
+									Type:     schema.TypeList,
```

Comment:
is this a safe change to make or is it somehow breaking?

Reply:
this should be fine providing we were already returning the values in an order. the `TypeSet` difference really comes into play when we are talking about plan and diff evaluation.